helper/funcs_test.go

```diff
@@ -34,6 +34,27 @@ func TestSliceStringContains(t *testing.T) {
 	require.False(t, SliceStringContains(list, "d"))
 }
 
+func TestSliceStringHasPrefix(t *testing.T) {
+	list := []string{"abc", "def", "abcdef", "definitely", "most definitely"}
+	require.True(t, SliceStringHasPrefix(list, "a"))
+	require.False(t, SliceStringHasPrefix(list, "b"))
+	require.False(t, SliceStringHasPrefix(list, "c"))
+	require.True(t, SliceStringHasPrefix(list, "d"))
+	require.True(t, SliceStringHasPrefix(list, "mos"))
+	require.True(t, SliceStringHasPrefix(list, "def"))
+}
+
+func TestStringHasPrefixInSlice(t *testing.T) {
+	prefixes := []string{"a", "b", "c", "definitely", "most definitely"}
+	require.True(t, StringHasPrefixInSlice("alpha", prefixes))
+	require.True(t, StringHasPrefixInSlice("bravo", prefixes))
+	require.True(t, StringHasPrefixInSlice("charlie", prefixes))
+	require.False(t, StringHasPrefixInSlice("delta", prefixes))
+	require.True(t, StringHasPrefixInSlice("definitely", prefixes))
+	require.False(t, StringHasPrefixInSlice("most", prefixes))
+	require.True(t, StringHasPrefixInSlice("most definitely", prefixes))
```

Comment:
I'm finding it hard to digest the tests and expected results. Separating the True from false cases would help.

Reply:
I separated the true/false cases, used common elements between the two tests, and added comments to help improve readability.